host/host.cpp

```diff
@@ -6,80 +6,89 @@
 #include <openenclave/host.h>
 #include <thrift/protocol/TBinaryProtocol.h>
 #include <thrift/server/TSimpleServer.h>
-#include <thrift/transport/TServerSocket.h>
 #include <thrift/transport/TBufferTransports.h>
+#include <thrift/transport/TServerSocket.h>
 
-#include "../errors/errors.h"
-#include "ortoa_u.h"
-#include "redis.h"
 #include "../constants/constants.h"
 #include "../constants/shared.h"
```

Comment:
One thing I noticed is that our includes sometimes try to use relative path and sometimes not (ex. `#include "../constants/shared.h"` vs  `#include "constants/shared.h"`). This was remnants of our old file directory structure + Makefile usage. Would it be possible to switch to one method or the other with CMake?